src/CollAction/Models/ProjectViewModels/DisplayProjectViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace CollAction.Models
+{
+    public class DisplayProjectViewModel
+    {
+        public Project Project;
```

Comment:
A small style issue, use:
public Project Project { get; set; }
for this, a shorthand for a getter. If you make it a property, then it's possible to change the implementation and storage of the property without changing the interface.

It doesn't matter terribly much for viewmodel classes, still, good habit to get into. https://msdn.microsoft.com/en-us/library/x9fsa0sw.aspx